arch/arm/boot/dts/overlays/revpi-connect-overlay.dts

```diff
@@ -284,6 +284,9 @@
 			pinctrl-0 = <&rs485_pins>;
 			linux,rs485-enabled-at-boot-time;
 			status = "okay";
+			pi_bridge {
```

Comment:
Missing new line here or one newline to much in the other overlay (line 262).  Sorry for nitpicking, just want to have this consistent  :smile: